.github/workflows/sync-files-to-repos.yml

```diff
@@ -0,0 +1,22 @@
+# Sync common files to other repos.
+# Adjust which files are synced in the .github/sync.yml file.
+
+name: Sync Files to repos
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      - name: Run GitHub File Sync
+        uses: BetaHuhn/repo-file-sync-action@v1
```

Comment:
We should probably use the commit sha here instead of the version. Considering we're providing the action with a fairly permissive PAT, we should be extra careful not relying on overridable tags.

Reply:
We definitely want to use a commit sha here instead of the `v1` tag. Any 3rd-party action can be taken over and a tag overridden - since we pass a write-capable PAT to this action we need to make sure we guard ourselves against that.

Reply:
updated